scripts/variantstore/wdl/test/GvsQuickstartIntegration.wdl

```diff
@@ -36,6 +40,7 @@ workflow GvsQuickstartIntegration {
     File full_exome_interval_list = "gs://gcp-public-data--broad-references/hg38/v0/bge_exome_calling_regions.v1.1.interval_list"
     String expected_subdir = if (!chr20_X_Y_only) then "all_chrs/"  else ""
     File expected_output_prefix = "gs://gvs-internal-quickstart/integration/2024-10-29/" + expected_subdir
+    File truth_data_prefix = "gs://gvs-internal-quickstart/integration/test_data/2025-01-17/"
```

Comment:
can you set this value in your method config during testing to avoid accidentally committing like this?